frontend/src/scenes/data-warehouse/settings/source/Schemas.tsx

```diff
@@ -70,14 +70,19 @@ export const SchemaTable = ({ schemas, isLoading }: SchemaTableProps): JSX.Eleme
                             return (
                                 <LemonSelect
                                     className="my-1"
-                                    value={schema.sync_frequency || 'day'}
+                                    value={schema.sync_frequency || '6hour'}
```

Comment:
why faster by default?

Reply:
I feel like 1-day is too long as a default. 6-hours feels right. But importantly, this is editable